File: Level_1/problem_1158.java
import java.util.ArrayList;
import java.util.Scanner;

public class Problem_1158 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        ArrayList<Integer> total_somas = new ArrayList<>();
        for (int i = 0; i < n; i++){
            int x = sc.nextInt();
            int y = sc.nextInt();
            int soma = 0;
            if (x % 2 == 0){
                x += 1;
            }
            for (int j = 0; j < y; j++){
                soma += x;
                x += 2;
            }
            total_somas.add(soma);
        }sc.close();
        for (int somas : total_somas){
            System.out.println(somas);
        }
    }
}

File: Level_1/problem_1158.py
n = int(input())

total_somas = []

for i in range(n):
    x, y = map(int, input().split())
    soma = 0
    if x % 2 == 0:
        x += 1
    for j in range(y):
        soma += x
        x += 2
    total_somas.append(soma)

for soma in total_somas:
    print(soma)
